results/pathway/hgsc/Project_node38genes_neg/enrichment_results_node38genes_neg.txt

```diff
@@ -0,0 +1,17 @@
+geneset	description	link	C	O	E	R	PValue	FDR	overlapGene	OverlapGene_UserID
```

Comment:
What does C O E and R stand for? I notice not every GO has a value for these fields.

Reply:
These files are autogenerated - I would not review them. They are included for posterity's sake!

Reply:
anyway, I believe `C` stands for `count` or how many genes are in the pathway; `O` stands for `observed` for how many input genes were assigned to the given pathway; `E` stands for expected number of genes observed under random assignment. I am not sure what `R` is